src/service/print/lang/english.json

```diff
@@ -0,0 +1,9 @@
+{
+    "studentName" : "Student Name",
```

Comment:
These files should be taken from blob rather than adding to the code repo. Any new language should not require any code code, it will be configuration driven.

Reply:
Where must i save these language files and how we will be using them. If there is any reference, please share.

Reply:
@vaivk369 , please help me with this if you can.